### PR TITLE
Graph theory - minor improvements of Huneke's proof

### PR DESCRIPTION
**Pull request by avekens**
* ~ elinsn added
* comment of ~df-fusgr adjusted
* ~frcond3 split into 2 theorems
* lemmata for ~frgrncvvdeq renamed and optimized
* comments of ~frgrwopreg and its lemmata adjusted

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -50103,6 +50103,12 @@ we define it to be empty in this case (see ~ opprc1 , ~ opprc2 , and
     DEFZADBGZCGZHZIZADBIZGZADCIZGZHZADBCJZIUCUEJRUBADSIZADTIZHUGADSTKRUIUDUJUFA
     DBELADCELMNADUHUABCOPUCUEOQ $.
 
+  $( If the intersection of two classes is a (proper) singleton, the singleton
```
Comment (benjub) [inline set.mm:50106]: ", then the" (not mandatory, though)

File: set.mm [hunk 2]
```diff
@@ -424182,11 +424182,12 @@ then they have the same degree (i.e., the same number of adjacent
...
 
-    $( In a friendship graph there are either no vertices or exactly one vertex
-       having degree K, or all or all except one vertices have degree K.
-       TODO-AV: proof can be shortened by using ~ bj-mp2d after it is moved to
-       main set.mm.  (Contributed by Alexander van der Vekens, 31-Dec-2017.)
-       (Revised by AV, 10-May-2021.) $)
+    $( In a friendship graph there are either no vertices ( ` A = (/) ` ) or
+       exactly one vertex ( ` ( # `` A ) = 1 ` ) having degree ` K ` , or all
+       ( ` B = (/) ` ) or all except one vertices ( ` ( # `` B ) = 1 ` ) have
+       degree ` K ` .  TODO-AV: proof can be shortened by using ~ bj-mp2d after
```
Comment (benjub) [inline set.mm:424188]: I'll try a minimization when I have time, to see how many proofs can be shortened using bj-mp2d.

**Review by benjub — APPROVED** [review]

File: set.mm (hunk 1 above)
Comment (jkingdon) [reply set.mm:50106]: Either wording reads fine to me :shrug:

**Review by jkingdon — APPROVED** [review]

File: set.mm (hunk 1 above)
Comment (avekens) [reply set.mm:50106]: I will change this in my next PR.

File: set.mm (hunk 2 above)
Comment (avekens) [reply set.mm:424188]: The proof of this theorem will be shortened in my next PR anyway. Another shortening by using ~ bj-mp2d will not be possible anymore.

**Pull request merged**